src/main.rs

```diff
@@ -295,6 +295,10 @@ pub struct TopLevel {
     #[argh(switch, short = 'V', long = "version")]
     /// print version information
     version: bool,
+
+    #[argh(option, long = "igvm")]
+    /// path to IGVM file
+    igvm: Option<String>,
```

Comment:
Please put it behind the `igvm` feature gate.

Reply:
FYI: there is a ongoing PR about switching back to using `clap`. So be prepared for a rebase shortly.

Reply:
IGVM is similarly loaded into memory for KVM and TDX, We tested the boot on KVM. QEMU does not support it. We do not have plan to support for KVM atm. But anyone wnats to add the feature we are happy to help. Our initial plan is to add IGVM and SNP support on MSHV.